src/PHPDebt/Console/Command/PHPDebtCommand.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace PHPDebt\Console\Command;
+
+use PHP_CodeSniffer\Config;
+use PHP_CodeSniffer\Runner;
+use PHPDebt\PHPDebtMD;
+use SebastianBergmann\FinderFacade\FinderFacade;
+use SebastianBergmann\PHPLOC\Analyser;
+use Symfony\Component\Console\{
+	Command\Command,
+	Input\InputArgument,
+	Input\InputInterface,
+	Input\InputOption,
+	Output\OutputInterface
+};
+
+/**
+ * The class for all `phpdebt` command.
+ *
+ * @package PHPDebt\Console\Command
+ */
+class PHPDebtCommand extends Command {
+
+	/**
+	 * {@inheritdoc}
+	 */
+	protected function configure() {
+		$this->setName('phpdebt')
+			->addArgument('path', InputArgument::REQUIRED, 'The path of the file or directory that needs to be scanned.')
+			->addOption('standard', 's', InputOption::VALUE_REQUIRED, 'The path of the phpcs.xml.dist')
+			->setDescription('This tool provides code quality score based on a number of standards from existing code analysis tools.');
+	}
+
+	/**
+	 * {@inheritdoc}
+	 */
+	protected function execute(InputInterface $input, OutputInterface $output) {
+		$code_faults = 0;
+		$standards_faults = 0;
+		$path = $input->getArgument('path');
+		$option = $input->getOption('standard');
+
+		$debt_md = new PHPDebtMD();
+		$code_faults += $debt_md->process($path, 'cleancode');
+		$code_faults += $debt_md->process($path, 'codesize');
+		$code_faults += $debt_md->process($path, 'design');
+		$code_faults += $debt_md->process($path, 'naming');
+		$code_faults += $debt_md->process($path, 'unusedcode');
+
+		$runner = new Runner();
+		// If PHPCS config is provided, then execute this.
+		if ($option) {
+			define('PHP_CODESNIFFER_CBF', FALSE);
+			$runner->config = new Config(["--standard=$option"]);
```

Comment:
Instead of having this if be running phpcs in two contexts, we should move the running until after and have the if just for determining if we need a default for `$option`.
I mean to say:
```
if (!$option) {
  $option = 'sane default';
}
$runner->config = new Config(["--standard=$option"]);
...
```